railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -82,6 +82,10 @@ def dockerfiles
       chmod "bin/docker-entrypoint", 0755 & ~File.umask, verbose: false
     end
 
+    def rubocop
+      template ".rubocop.yml"
```

Comment:
Should follow the other dot files and use: `template "rubocop.yml", ".rubocop.yml"`.